carts/views.py

```diff
@@ -0,0 +1,50 @@
+import json
+
+from django.http           import JsonResponse
+from django.views          import View
```

Comment:
Line 4와 line5 사이에 한 줄 띄워주세요
```suggestion
from django.views          import View

```